docs/upgrade-notes.asciidoc

```diff
@@ -48,6 +48,42 @@ For Elastic Security solution release information, refer to {security-guide}/rel
 [float]
 === Breaking changes
 
+[discrete]
+[[breaking-207091]]
+.Removed legacy security rules bulk endpoints (9.0.0)
+[%collapsible]
+====
+*Details* +
+--
+* `POST /api/detection_engine/rules/_bulk_create` has been replaced by `POST /api/detection_engine/rules/_import`
+* `PUT /api/detection_engine/rules/_bulk_update` has been replaced by `POST /api/detection_engine/rules/_bulk_action`
+* `PATCH /api/detection_engine/rules/_bulk_update has been replaced by `POST /api/detection_engine/rules/_bulk_action`
+* `DELETE /api/detection_engine/rules/_bulk_delete` has been replaced by `POST /api/detection_engine/rules/_bulk_action`
+* `POST api/detection_engine/rules/_bulk_delete` has been replaced by `POST /api/detection_engine/rules/_bulk_action`
+--
+These changes were introduced in PR {kibana-pull}#197422[#197422]
+
+*Impact* +
+Deprecated endpoints will fail with a 404 status code starting from version 9.0.0
+
+*Action* +
+--
+Update your implementations to use the new endpoints:
+
+1. **For bulk creation of rules:**
+   - Use `POST /api/detection_engine/rules/_import` (link:https://www.elastic.co/docs/api/doc/kibana/v8/operation/operation-importrules[API documentation]) to create multiple rules along with their associated entities (e.g., exceptions and action connectors).
+   - Alternatively, create rules individually using `POST /api/detection_engine/rules` (link:https://www.elastic.co/docs/api/doc/kibana/v8/operation/operation-createrule[API documentation]).
+
+2. **For bulk updates of rules:**
+   - Use `POST /api/detection_engine/rules/_bulk_action` (link:https://www.elastic.co/docs/api/doc/kibana/v8/operation/operation-performrulesbulkaction[API documentation]) to update fields in multiple rules simultaneously.
+   - Alternatively, update rules individually using `PUT /api/detection_engine/rules` (link:https://www.elastic.co/docs/api/doc/kibana/v8/operation/operation-updaterule[API documentation]).
+
+3. **For bulk deletion of rules:**
+   - Use `POST /api/detection_engine/rules/_bulk_action` (link:https://www.elastic.co/docs/api/doc/kibana/v8/operation/operation-performrulesbulkaction[API documentation]) to delete multiple rules by IDs or query.
+   - Alternatively, delete rules individually using `DELETE /api/detection_engine/rules` (link:https://www.elastic.co/docs/api/doc/kibana/v8/operation/operation-deleterule[API documentation]).
```

Comment:
@natasha-moore-elastic this link is improperly resolving to https://www.elastic.co/docs/api/doc/kibana/v8/operation/operation-update-package-policy in the preview.

Reply:
Same as above.